app/actions/websocket/channel.ts

```diff
@@ -423,10 +423,10 @@ export async function handleChannelDeletedEvent(serverUrl: string, msg: WebSocke
                 await popToRoot();
 
                 if (await isTablet()) {
-                    const channelToJumpTo = await getNthLastChannelFromTeam(database, currentChannel?.teamId);
+                    const channelToJumpTo = await getNthLastChannelFromTeam(database, currentChannel.teamId);
```

Comment:
if `currentChannel` is a DM/GM it won't have a team Id, this should fallback to `currentTeamId`.

Same applies to the other function modified in this file